packages/table/src/table.tsx

```diff
@@ -51,6 +53,14 @@ export interface ITableProps extends IProps, IRowHeights, IColumnWidths {
     fillBodyWithGhostCells?: boolean;
 
     /**
+     * If exists, a callback that returns the data for a specific cell. This need not
+     * match the value displayed in the `<Cell>` component. The value will be
+     * invisibly added as `textContent` into the DOM before copying. If not exists,
+     * copy via hotkeys (mod+c) will not work.
```

Comment:
This works the same as it does for the context menu copy -- you need to give that a callback too. The reason why is what you want rendered (with divs and styles and pretty things) may be radically different from what you want copied (the data). I've talked to @themadcreator at length about this, and this seems like the best way :-/